cli/restart.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+// restartDescription is used to describe restart command in detail and auto generate command doc.
+var restartDescription = "restart one or more containers"
+
+// restartCommand uses to implement 'restart' command, it restarts a container.
```

Comment:
s/restartCommand/RestartCommand
BTW, as your restartDescription says `restart one or more containers`, so this comment should change